docs/articles/documentation/why-testcafe/README.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: docs
+title: Why TestCafe?
+permalink: /documentation/why-testcafe/
+---
+
+# Why TestCafe?
+
+TestCafe excels where other end-to-end testing solutions fall short.
+
+## One NPM package
+
+TestCafe doesn't require the use of extra tools and frameworks. All you need to get up and running is a recent version of Node.js, a browser — any browser, and the TestCafe NPM package.
+
+## As simple and extendable as JavaScript itself
+
+TestCafe tests are full-fledged Node.js apps. They are compatible with your favourite JavaScript libraries and preprocessors.
+
+## Complex tests made easy
+
+TestCafe’s intuitive and powerful toolkit makes testing easy, whether you’re working with a landing page or a complex web app.
```

Comment:
This sentence doesn't really say "why"... The words "powerful and intuitive" need explanation, a backup.  It's better to always back up your claims. You may not fully describe how something works, but at least mention the thing that makes something powerful or easy. 

Here's an attempt to inject a reason (I don't know that this reason is correct though :-) )

TestCafe API doesn't limit your interactions with the page - you can always do more with JavaScript. That's why you can test any page - from static landing pages, to entry forms, to complex web apps.

Reply:
I guess these are introductory phrases, proofs follow within this section. If I were to insert a reason right here, I'd say:

TestCafe is ready for advanced scenarios with iframes, multiple windows, security-sensitive features, etc. That's why you can test any page - from static landing pages, to entry forms, to complex web apps.